### DISCUSSION TITLE
Cannot enter the official imgaeMagick website

### DISCUSSION THREAD

**Discussion by SetonLiang** [ImageMagick/ImageMagick#7494] [Help]
### ImageMagick version

6.2.7

### Operating system, version and so on

Window11

### Description

![image](https://github.com/user-attachments/assets/79f89c08-9b5b-4142-98b3-4416b7e0f69c)
I want to download this source because my computer owns the Window system. However，I cannot enter the official website.

**Comment by dlemstra** [comment]
As we mentioned in other issues/discussions the website will be available again in a couple days.

**Reply by SetonLiang** [reply]
Does it mean that the website is under maintenance now? I wondor how many days will it take.

**Reply by dlemstra** [reply]
It will take as long as it takes. It's down due to power issues.

**Reply by SetonLiang** [reply]
Okay,thanks.
